### PR TITLE
Java go lang differential evaluation

### PR DESCRIPTION
**Pull request by abahmanem**
Differential evaluation of SDKs for required compliance - Java and Golang

### CODE REVIEW

File: Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| chain_get_state_root_hash |        [source](https://github.com/syntifi/casper-sdk/blob/main/src/main/java/com/syntifi/casper/sdk/service/CasperService.java#L98)   | ✅  |             |
+| state_get_account_info    |        [source](https://github.com/syntifi/casper-sdk/blob/main/src/main/java/com/syntifi/casper/sdk/service/CasperService.java#L149)                                                                | ✅  | |
+| state_get_balance         |        [source](https://github.com/syntifi/casper-sdk/blob/main/src/main/java/com/syntifi/casper/sdk/service/CasperService.java#L182)        | ✅  |             |
+| state_get_dictionary_item |        [source](https://github.com/syntifi/casper-sdk/blob/main/src/main/java/com/syntifi/casper/sdk/service/CasperService.java#L171)  | ✅  |             |
+| query_global_state        |        N/A        | ❌  | Est. : 4 hours|
+
+
+## Reviewer notes
+
+* There are one missing method : query_global_state. The estimated implementation time for this method is 4 hours.
```
Comment (mrkara) [inline Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md:23]: "There are one missing method" -> "There is one missing method"

File: Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* The implementation for chain_get_block should be merged into a single function and take BlockIdentifier param. The estimated update time for this merge is 1 hour.
+* Others methods and types are aligned with the SDK standard.
```
Comment (mrkara) [inline Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md:25]: "Others methods" -> "Other methods"

File: Reviews/ 359 - Improved Golang SDK for Casper Network/Differential evaluation of GoLang SDK/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+| state_get_balance         |        [source](https://github.com/casper-ecosystem/casper-golang-sdk/blob/main/sdk/rpc_client.go#L67)         | ✅  |             |
+| state_get_dictionary_item |        N/A        | ❌  | Est.: 4 hours|
+| query_global_state        |        N/A        | ❌  | Est.: 4 hours|
+
+
+## Reviewer notes
+
+* There are three missing methods :  state_get_account_info, state_get_dictionary_item  and query_global_state. The estimated implementation time for each method is 4 hours.
+* The implementation for chain_get_block should be merged into a single function and take BlockIdentifier param. The estimated update time for this merge is 1 hour.
+* Others methods and types are aligned with the SDK standard.
```
Comment (mrkara) [inline Reviews/ 359 - Improved Golang SDK for Casper Network/Differential evaluation of GoLang SDK/README.md:24]: "Others methods" -> "Other methods"

**Review by mrkara — CHANGES_REQUESTED** [review]

File: Reviews/ 359 - Improved Golang SDK for Casper Network/Differential evaluation of GoLang SDK/README.md (hunk 3 above)
Comment (abahmanem) [reply Reviews/ 359 - Improved Golang SDK for Casper Network/Differential evaluation of GoLang SDK/README.md]: Sorry for the typos. done

File: Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md (hunk 1 above)
Comment (abahmanem) [reply Reviews/186 - Java SDK Casper/Differential evaluation of Java SDK/README.md]: done

**Comment by ggurbet** [comment]
Cleanup: This PR is no longer relevant, it can be closed by an admin.

**Pull request closed without merging**